models/product/src/product/product-draft/presets/sample-data-fashion/anniversary-shirt.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { TProductDraft } from '../../../types';
+import anniversaryShirt from './anniversary-shirt';
+
+describe(`with anniversaryShirt preset`, () => {
+  it('should return a sample anniversaryShirt product preset', () => {
```

Comment:
nit: inadvertent camelcasing? `sample anniversaryShirt`

Several other files have this